Snowdog_Components/components/Atoms/typography/heading/heading.config.js

```diff
@@ -146,5 +146,29 @@ module.exports = {
         text: 'Sixth level heading - tag and class'
       }
     },
+    {
+      name: 'page-class',
```

Comment:
I think this would fit the naming convention better:
```suggestion
      name: 'class-page',
```